terraform/Makefile

```diff
@@ -0,0 +1,43 @@
+##TASKS
+init:
+	bash -x ../scripts/terraform-auto/init.sh
```

Comment:
We should use 3M pattern here so the terraform commands should be executed from a docker container.
The reason is that the terraform cli might not be available in the github actions and also to ensure that the same terraform CLI version will be used to execute the commands.

Sample code:

```bash
DOCKER_RUNNER?=docker-compose run --rm

tf-ci-plan:
	@$(DOCKER_RUNNER) ci-terraform init
	@$(DOCKER_RUNNER) ci-terraform plan
.PHONY:tf-ci-plan
```

you will also need to add the `ci-terraform` service in the docker-compose.yml:

```yaml
  ci-terraform:
    image: hashicorp/terraform:0.12.29
    working_dir: /terraform
    volumes:
      - ${PWD}/terraform:/terraform
    environment:
      - AWS_ACCESS_KEY_ID
      - AWS_SECRET_ACCESS_KEY
      - TF_IN_AUTOMATION=yes
```

Reply:
done